projects/core/src/state/utils/scoped-loader/entity-scoped-loader.actions.ts

```diff
@@ -86,13 +89,32 @@ export namespace EntityScopedLoaderActions {
     implements EntityScopedLoaderAction, ErrorAction
   {
     type = ENTITY_FAIL_ACTION;
-    error: ErrorActionType;
+    error: ActionErrorProperty;
     readonly meta: EntityScopedLoaderMeta;
 
+    /**
+     * @deprecated Please use `error` parameter other than `null` or `undefined`.
+     *
+     *             Note: Allowing for `null` or `undefined` will be removed in future versions
+     *             together with the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+     **/
+    constructor(
+      entityType: string,
+      id: string | string[],
+      error: null | undefined,
+      scope?: string
+    );
+    constructor(
```

Comment:
`query:` Is the order od constructor's overriding proper? Shouldn't the new, not deprecated definition, be first?